src/QMCDrivers/WFOpt/QMCFixedSampleLinearOptimizeBatched.h

```diff
@@ -51,16 +51,16 @@ class QMCFixedSampleLinearOptimizeBatched : public QMCLinearOptimizeBatched,
                                       Communicate* comm);
 
   ///Destructor
-  ~QMCFixedSampleLinearOptimizeBatched();
+  virtual ~QMCFixedSampleLinearOptimizeBatched();
```

Comment:
Is virtual necessary?

Reply:
Probably not.  It was a change I made to try fix the Rng crash issue.